cmd/kubeless/function.go

```diff
@@ -117,41 +119,72 @@ func getFileSha256(file string) (string, error) {
 	return checksum, err
 }
 
-func uploadFunction(file string, cli kubernetes.Interface) (string, string, string, error) {
-	var function, contentType, checksum string
-	stats, err := os.Stat(file)
+func isMinioAvailable(cli kubernetes.Interface) bool {
+	_, err := cli.Core().Services("kubeless").Get("minio", metav1.GetOptions{})
 	if err != nil {
-		return "", "", "", err
+		return false
```

Comment:
I think we do. If it cannot read the file or it cannot retrieve a certain resource (when it should be there) we should abort/disable the process, if not it will lead to weird behaviors.